### PR TITLE
Create maven.yml

### LINKED ISSUE #14: Add continuous testing
When you create a PR, it's extremely useful to know if it's going to break tests. GitHub supports this through "workflows". Setup a GitHub continuous testing workflow.

### PR DESCRIPTION
**Pull request by kanbi18**
Implementing continuous testing on java files

### CODE REVIEW

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Build with Maven
+      run: mvn -B package --file pom.xml
```
Comment (freeboson) [inline .github/workflows/maven.yml]: For some reason, we don't have `pom.xml` in the root directory, it's `maptivist/pom.xml`. The script should either do (something like) `cd maptivist && mvn ...` or maybe you can just do `mvn -B package --file maptivist/pom.xml`.

**Review by freeboson — CHANGES_REQUESTED** [review]
This workflow is failing (see the big red cross above), because our `pom.xml` is not where it should be.

File: .github/workflows/maven.yml (hunk 1 above)
Comment (freeboson) [inline .github/workflows/maven.yml]: Also, add a step to actually do the testing, it'll be something like:

```
    - name: Test with Maven
      run: mvn test ....
```

**Review by TBarrett553 — APPROVED** [review]
Thanks for setting this up!

**Review by freeboson — APPROVED** [review]
LGTM

**Pull request merged**